projects/lights_control/inc/lights_events.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+typedef enum {
+  // front board events
+  EVENT_HORN = 0,
```

Comment:
Please see style-guide for `enum` naming.

You can refer to [`chaos_events.h`](https://github.com/uw-midsun/firmware/blob/master/projects/chaos/inc/chaos_events.h) for additional examples.

Reply:
+1 you should probably group your events into separate enums as it is easier to do guard clauses when you have defined ranges for a type of event.

Reply:
cool I fixed it. 
For the record though, I was following `input_event.h` in `driver_controls`. It seems to me that `InputEvent` is also a generic name used in there, and something like `DriverControlsEvent` would be more appropriate.

Reply:
@ckitagawa do you mean separate enums, with one enum starting with the end value of the other enum? something like: 
```
typedef enum {
//
  SIGNAL_RIGHT = 0,
  SIGNAL_LEFT,
//
  NUM_SIGNAL_EVENTS=3
} LightsSignalEvents
typedef enum {
  LIGHTS_HIGHBEAMS = 3,
  LIGHTS_LOWBEAMS,
// etc
} LightsHeadlightEvents
```
because otherwise the events will overlap. For example `0` will refer to the first event of `LightsHeadlightEvents` and also the first event of `LightsSignalEvents` and that's kind of problematic.  But this would also have a problem because I'd have to manually set the NUM_ENUM field at the end of each enum to contain the right number of that specific enum type.

Reply:
Driver controls is being reworked. Don't use it as an example.

In its current form, `InputEvent` is accurate because it refers to events from external inputs. It is not a generic name.